src/main/java/io/fabric8/jenkins/openshiftsync/BuildConfigWatcher.java

```diff
@@ -141,15 +155,25 @@ private void upsertJob(final BuildConfig buildConfig) throws Exception {
         @Override
         public Void call() throws Exception {
           String jobName = jenkinsJobName(buildConfig);
+          String jobFullName = jenkinsJobFullName(buildConfig);
           WorkflowJob job = getJobFromBuildConfig(buildConfig);
+          Jenkins activeInstance = Jenkins.getActiveInstance();
+          ItemGroup parent = activeInstance;
+          if (job == null) {
+            job = (WorkflowJob) activeInstance.getItemByFullName(jobFullName);
+          }
           boolean newJob = job == null;
           if (newJob) {
-            job = new WorkflowJob(Jenkins.getActiveInstance(), jobName);
+            String disableOn = getAnnotation(buildConfig, DISABLE_SYNC_CREATE);
```

Comment:
i don't see anything that sets this.  were you intending to set this on the buildconfigs being created from the sync plugin?